### DISCUSSION TITLE
how to enrich alert using ingest pipeline on cross-cluster  es

### DISCUSSION THREAD

**Discussion by ppd520** [Security-Onion-Solutions/securityonion#9762] [Unsupported Versions]
hi，

I built the manager node and search node, but now I need to expand the alarm data with  business information according to the destination IP; Try to establish the source index on the manager node and create the enrich policy; But when search node ingest with enrich process, an error is reported: no enrich index exists for policy with name [xxxxx]

Do I need to create a source index on the search node? After the source is updated, do all search nodes need to be updated?

**Comment by cm-ops** [comment]
Which data are looking to enrich and where is that data located (manager or search)?
